app/controllers/admin/events_controller.rb

```diff
@@ -7,6 +7,19 @@ def index
     @events = Event.order(updated_at: :desc)
   end
 
+  def new
+    @event = Event.new
+  end
+
+  def create
```

Comment:
looks good, but how does this handle error messages? 

I've been using the format:
def create
    all the event stuff except save
    if @event.save
        redirect_to admin_events_path
    else
        render 'form'
    end
end

which allows for the form to regenerate with error messages and all of the content that had been typed in. Is that working anyway?